gandi/cli/tests/commands/test_contact.py

```diff
@@ -152,5 +152,7 @@ def test_create_apikey_ok(self):
 Will save your apikey into the config file.""")
         self.assertEqual(result.exit_code, 0)
 
-        self.assertEqual(GandiModule._conffiles,
```

Comment:
In previous implementation, I used to create a default `editor` config value, and had to update this test.
I then understood how wrong I was, updated the code, but forgot to revert the change to this test.
Doing it now.